apparmor.d/profiles-s-z/xauth

```diff
@@ -31,11 +31,11 @@ profile xauth @{exec_path} {
   owner /tmp/serverauth.*-n rw,
   owner /tmp/serverauth.*   rwl -> /tmp/serverauth.*-n,
 
-  owner /tmp/runtime-*/xauth_?????? r,
+  owner /tmp/runtime-*/xauth_@{rand6} r,
 
-  owner @{run}/user/@{uid}/xauth_?????? rw,
-  owner @{run}/user/@{uid}/xauth_??????-c w,
-  owner @{run}/user/@{uid}/xauth_??????-l wl,
```

Comment:
On a second thought, these might cause problems, because I expanded ambiguous `/xauth_*` to unequivocal `/xauth_@{rand6}` in other profiles.